packages/chips/ChipCheckmark.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+
+export default class ChipCheckmark extends Component {
+  width = null;
+
+  init = (element) => {
+    // The checkmark's width may initially be set to 0, so use the checkmark's height as a proxy since the
+    // checkmark should always be square.
+    this.width = element.getBoundingClientRect().height;
```

Comment:
have you tried unmounting this component? the `ref` is called with `null` on unmount. So it might throw a null error here.